Orange/widgets/regression/owsgdregression.py

```diff
@@ -42,7 +40,7 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     learning_rate = settings.Setting(InvScaling)
 
     want_main_area = False
-    resizing_enabled = False
+    resizing_enabled = True
```

Comment:
Yes, the reporting function here is fine! (Save for a few minor issues as noted in the other comments.)